docs/source/interfaces/ISIS SANS v2.rst

```diff
@@ -284,7 +284,7 @@ Masking information
 
 The masking table shows detailed information about the masks which will be applied.
 These masks include bin masks, cylinder masks, mask files, spectrum masks, angle masks
-and masks for the beam stop. If as mask is applied only to a particular detector
+and line masks for the beam stop arm. If a mask is applied only to a particular detector
 then this will be shown in the masking table. Note that data needs to be specified
```

Comment:
I determined there was no reference to the beam stop arm, by searching for 'arm', which only occurred in LARMOUR.